<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployePages</class>
 <widget class="QWidget" name="EmployePages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>554</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="topBarLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnListEmployees">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Liste des Employes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddEmployee">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Ajouter un Employe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStatistics">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Statistiques</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerTop">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnEmail">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="pagesStack">
     <widget class="QWidget" name="pageList">
      <layout class="QVBoxLayout" name="listLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="searchLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="searchBox">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>40</height>
            </size>
           </property>
           <property name="placeholderText">
                <string>Search employees...</string>
           </property>
          </widget>
         </item>
            <item>
             <widget class="QLabel" name="lblSortBy">
              <property name="text">
                <string>Sort by:</string>
              </property>
             </widget>
            </item>
         <item>
          <widget class="QComboBox" name="sortCombo">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>40</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnFilter">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacerSearch">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnGeneratePdf">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Generate PDF File</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="employeeTable"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageForm">
      <layout class="QVBoxLayout" name="formLayout">
       <item>
        <widget class="QLabel" name="formTitle">
         <property name="text">
          <string>Employee Form</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="formGrid">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <property name="horizontalSpacing">
          <number>15</number>
         </property>
         <property name="verticalSpacing">
          <number>6</number>
         </property>
                 <item row="0" column="0">
                    <widget class="QLabel" name="lblCin">
                     <property name="text">
                        <string>CIN</string>
                     </property>
                    </widget>
                 </item>
                 <item row="0" column="1">
                    <widget class="QLabel" name="lblType">
                     <property name="text">
                        <string>Type</string>
                     </property>
                    </widget>
                 </item>
                 <item row="1" column="0">
                    <widget class="QLineEdit" name="inputCin">
                     <property name="placeholderText">
                        <string>e.g. 12345678</string>
                     </property>
                    </widget>
                 </item>
                 <item row="1" column="1">
                    <widget class="QComboBox" name="inputType"/>
                 </item>

                 <item row="2" column="0">
                    <widget class="QLabel" name="lblFirstName">
                     <property name="text">
                        <string>First Name</string>
                     </property>
                    </widget>
                 </item>
                 <item row="2" column="1">
                    <widget class="QLabel" name="lblEnrollment">
                     <property name="text">
                        <string>Enrollment Date</string>
                     </property>
                    </widget>
                 </item>
                 <item row="3" column="0">
                    <widget class="QLineEdit" name="inputFirstName">
                     <property name="placeholderText">
                        <string>e.g. Jean</string>
                     </property>
                    </widget>
                 </item>
                 <item row="3" column="1">
                    <widget class="QLineEdit" name="inputEnrollment">
                     <property name="placeholderText">
                        <string>DD/MM/YYYY</string>
                     </property>
                    </widget>
                 </item>

                 <item row="4" column="0">
                    <widget class="QLabel" name="lblLastName">
                     <property name="text">
                        <string>Last Name</string>
                     </property>
                    </widget>
                 </item>
                 <item row="4" column="1">
                    <widget class="QLabel" name="lblPhoto">
                     <property name="text">
                        <string>Photo</string>
                     </property>
                    </widget>
                 </item>
                 <item row="5" column="0">
                    <widget class="QLineEdit" name="inputLastName">
                     <property name="placeholderText">
                        <string>e.g. Dupont</string>
                     </property>
                    </widget>
                 </item>
                 <item row="5" column="1">
                    <widget class="QPushButton" name="btnUploadImage">
                     <property name="text">
                        <string>Upload Image</string>
                     </property>
                    </widget>
                 </item>

                 <item row="6" column="0">
                    <widget class="QLabel" name="lblGender">
                     <property name="text">
                        <string>Gender</string>
                     </property>
                    </widget>
                 </item>
                 <item row="6" column="1">
                    <widget class="QLabel" name="lblAge">
                     <property name="text">
                        <string>Age</string>
                     </property>
                    </widget>
                 </item>
                 <item row="7" column="0">
                    <widget class="QComboBox" name="inputGender"/>
                 </item>
                 <item row="7" column="1">
                    <widget class="QLineEdit" name="inputAge">
                     <property name="placeholderText">
                        <string>e.g. 28</string>
                     </property>
                    </widget>
                 </item>

                 <item row="8" column="0">
                    <widget class="QLabel" name="lblEmail">
                     <property name="text">
                        <string>Email</string>
                     </property>
                    </widget>
                 </item>
                 <item row="8" column="1">
                    <widget class="QLabel" name="lblPhone">
                     <property name="text">
                        <string>Phone Number</string>
                     </property>
                    </widget>
                 </item>
                 <item row="9" column="0">
                    <widget class="QLineEdit" name="inputEmail">
                     <property name="placeholderText">
                        <string>name@example.com</string>
                     </property>
                    </widget>
                 </item>
                 <item row="9" column="1">
                    <widget class="QLineEdit" name="inputPhone">
                     <property name="placeholderText">
                        <string>e.g. +216 12 345 678</string>
                     </property>
                    </widget>
                 </item>

                 <item row="10" column="0" colspan="2">
                    <widget class="QLabel" name="lblProgress">
                     <property name="text">
                        <string>Progress Report</string>
                     </property>
                    </widget>
                 </item>
                 <item row="11" column="0" colspan="2">
                    <widget class="QTextEdit" name="inputProgress">
                     <property name="placeholderText">
                        <string>Write a short progress note...</string>
                     </property>
                    </widget>
                 </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="formButtonsLayout">
         <item>
          <spacer name="horizontalSpacerFormLeft">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnConfirmUpdate">
           <property name="text">
            <string>Confirm Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnConfirmAdd">
           <property name="text">
            <string>Confirm Add</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacerFormRight">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageStats">
      <layout class="QHBoxLayout" name="statsLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QWidget" name="statsLeftContainer" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="statsRightContainer" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
